packages/protobuf/src/messages.ts

```diff
@@ -1729,7 +1729,7 @@ export type ResetDevice = {
     u2f_counter?: number;
     skip_backup?: boolean;
     no_backup?: boolean;
-    backup_type?: string | number;
+    backup_type?: BackupType | Enum_BackupType;
```

Comment:
I assume this usage is here to allow for both:

`const x: BackupType = 'Bip39'`

and 

```ts
export const x1: Enum_BackupType = Enum_BackupType.Bip39

// Technically
export const x2: Enum_BackupType = 0
```


This seems a bit weird to me. 



In different place, you are deleting a comment 

```ts
// BackupType is a enum. in Features displayed as string, in resetDevice method param accepted as number
```

To me, this seems as a quite messy situation and I would suggest to split those types. One for `resetDevice` and other for `Features` array (with maybe mapping table between them)

Reply:
Yeah, unfortunately in Connect this is not unified at all. Some methods take the string, some the enum value, some both. 
In this case we should be able to limit the type to only the enum values in `resetDevice` and keep string for `Features`, generally we want to keep it as is in case of 3rd parties, but that's not really a problem here.

Reply:
> In this case we should be able to limit the type to only the enum values in resetDevice and keep string for Features

Lets do it then. :pray:

Reply:
Ok so is it alright now?